CreeDictionary/morphodict/templatetags/morphodict_orth.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+from django import template
+from django.utils.html import format_html
+
+from ..orthography import ORTHOGRAPHY
+
+register = template.Library()
+
+
+@register.simple_tag(name="orth", takes_context=True)
+def orth_tag(context, original_text: str) -> str:
+    """
+    Tag that generates a <span> with multiple orthographical representations
+    of the given text, in SRO. The inner text is determined by the
+    orth= cookie in the HTTP request.
+
+    e.g.,
+
+        {% orth 'wâpamew' %}
+
+    Yields:
+
+        <span lang="cr" data-orth
+              data-orth-latn="wâpamêw"
+              data-orth-latn-x-macron="wāpamēw"
+              data-orth-cans="ᐚᐸᒣᐤ">wâpamêw</span>
+    """
+    # Determine the currently requested orthography:
+    request_orth = context.request.COOKIES.get("orth", ORTHOGRAPHY.default)
+    return orth(original_text, orthography=request_orth)
+
+
+@register.filter
+def orth(original_text: str, orthography: str):
+    """
+    Filter that generates a <span> with multiple orthographical representations
+    of the given text.
+
+    e.g.,
+
+        {{ 'wâpamêw'|orth }}
+
+    Yields:
+
+        <span lang="cr" data-orth
+              data-orth-latn="wâpamêw"
+              data-orth-latn-x-macron="wāpamēw"
+              data-orth-cans="ᐚᐸᒣᐤ">wâpamêw</span>
+    """
+
+    conversions = {
+        code: ORTHOGRAPHY.converter[code](original_text)
+        for code in ORTHOGRAPHY.available
+    }
+    inner_text = conversions[orthography]
+    data_attributes = " ".join(f'data-orth-{code}="{{}}"' for code in conversions)
+    values = tuple(conversions.values()) + (inner_text,)
+
+    return format_html(
+        # TODO: do not hardcode lang="cr" here
+        '<span lang="cr" data-orth ' + data_attributes + ">{}</span>",
```

Comment:
The `lang="cr"` is the only part that was too lazy to make language-agnostic. I'm not entirely sure where this should be configured, as the language code is not just about orthography.

Reply:
AppConfig is the class that :
> """representing a Django application and its configuration."""

maybe each language specific django app need to have a `lang` attribute in there `apps.py`> `AppConfig` class

Then each orthography option in `settings.MORPHODICT_ORTHOGRAPHY['available']` should specify the apps it work for. So that the lang code can be found in the app config.

Or if we have the implicit assumption that each converting function works only for the language of the app, the current `settings.MORPHODICT_ORTHOGRAPHY` will be totally fine as the App can be extracted from the module path.

Reply:
Lol, I just made a new setting with the catchy name of `MORPHODICT_ISO_639_1_CODE`, since that's what it is.

I figure there would be a distinct `settings` module for each language that we support; I thinking having multiple languages support one `settings` file is just going to be a world of hurt :/